components/Inputs/Select/Select.vue

```diff
@@ -20,35 +22,61 @@
                 :id="associatedLabel"
                 ref="input"
                 :value="parsedInputValue"
-                :placeholder="placeholder"
+                :placeholder="placeholderValue"
                 :disabled="disabled"
                 :aria-label="label || 'no description'"
                 type="text"
                 readonly
                 @input="onValueChange"
                 @focus="onFocus"
                 @blur="onBlur">
-            <slot name="append">
+            <div
+                class="input__append">
+                <slot name="append" />
+                <ErrorHint
+                    v-if="isError"
+                    :hint="parsedErrorMessages" />
+                <InfoHint
+                    v-if="isDescription"
+                    :hint="description" />
                 <IconArrowDropDown :state="dropDownState" />
-            </slot>
+            </div>
         </div>
         <FadeTransition v-if="isMenuActive">
-            <SelectBaseContent
+            <DropDown
                 ref="menu"
                 :style="selectBoundingBox"
                 :fixed-content="fixedContentWidth">
                 <template #body>
                     <slot name="selectContent">
-                        <MultiselectListContent
-                            v-if="multiselect"
-                            :options="options"
-                            :selected-options="selectedOptions"
-                            @select="onSelectValue" />
-                        <SelectListContent
-                            v-else
-                            :options="options"
-                            :selected-option="String(value)"
-                            @select="onSelectValue" />
+                        <List>
+                            <template v-for="(option, index) in options">
+                                <slot
+                                    name="option"
+                                    :option="option"
+                                    :index="index">
+                                    <ListElement
+                                        v-if="isOptionValid(option)"
+                                        :key="index"
+                                        :small="small"
+                                        :selected="index === selectedOptionIndex"
+                                        @click.native="onSelectValue(option, index)">
+                                        <ListElementAction v-if="multiselect">
+                                            <CheckBox
+                                                :value="
+                                                    typeof selectedOptions[index] !== 'undefined'
```

Comment:
formatting

Reply:
What's wrong with that? The block is exceeding more than 100 lines, I need to make new line to keep readability - we have to implement solution for checking formatting along whole app, ya know, lines between etc.